clients/include/rocblas_test.hpp

```diff
@@ -113,11 +113,6 @@ class RocBLAS_TestName
     {
         // Placed inside function to avoid dependency on initialization order
         static table_t* table = test_cleanup::allocate<table_t>(&table);
-        if(!table)
-        {
-            fputs("Internal error: nullptr\n", stderr);
-            exit(EXIT_FAILURE);
-        }
```

Comment:
There's no need to test `table` for `nullptr` -- the `test_cleanup::allocate` already throws a `std::bad_alloc` exception if memory cannot be allocated. This is also a rare occurrence which should not need to be singled out. Removing this makes the code simpler.